bases/br_inep_indicadores_educacionais/curso_de_para_anual/table_config.yaml

```
@@ -0,0 +1,287 @@
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: ano_ingresso
+        description: Ano de ingresso do aluno no curso.
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: ano_referencia
+        description: Ano de referência do vínculo do ingressante.
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: prazo_integralizacao
+        description: Prazo mínimo de integralização de curso de graduação em número de anos
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: ano_integralizacao
+        description: Ano previsto de integralização do aluno ao curso.
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: prazo_acompanhamento
+        description: Prazo máximo de integralização de curso de graduação em número de anos
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: ano_max_acompanhamento
+        description: Ano máximo de acompanhamento da situação de vínculo do aluno
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: qtde_ingressante
+        description: Número de ingressantes do curso no ano de ingresso da coorte.
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: qtde_permanencia
+        description: Número de estudantes que permaneceram no curso de graduação no ano de referência da análise
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: qtde_concluinte
+        description: Número de estudantes que concluíram o curso de graduação no ano de referência da análise
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: qtde_desistencia
```

@Hevilyn não usamos `qtde` e sim `quantidade`, está na tabela de padrões BD+: https://docs.google.com/spreadsheets/d/1r4UqWx3d2jaRpoqp3ZkmJ07kerVP6KRO0unfchrOIuY/edit#gid=0
```suggestion
        name: quantidade_desistencia
```

---

editar nos demais campos